asciidoctorj-api/src/main/java/org/asciidoctor/log/Severity.java

```diff
@@ -1,10 +1,20 @@
 package org.asciidoctor.log;
 
 public enum Severity {
-    DEBUG,
-    INFO,
-    WARN,
-    ERROR,
-    FATAL,
-    UNKNOWN
+    DEBUG(0),
+    INFO(1),
+    WARN(2),
+    ERROR(3),
+    FATAL(4),
+    UNKNOWN(5);
+
+    private int rubyId;
```

Comment:
Oops, this must go away.
I thought I removed the rubyIds, somehow they sneaked in again.